sql/item_mods.sql

```diff
@@ -77499,6 +77499,7 @@ INSERT INTO `item_mods` VALUES (27970,31,26);    -- MEVA: 26
 INSERT INTO `item_mods` VALUES (27970,68,22);    -- EVA: 22
 INSERT INTO `item_mods` VALUES (27970,163,-500); -- DMGMAGIC: -500
 INSERT INTO `item_mods` VALUES (27970,384,400);  -- HASTE_GEAR: 400
+INSERT INTO `item_mods` VALUES (27970,485,10);    -- `SHIELD_MASTE`RY_TP: 10
```

Comment:
Align this properly and remove that stray ` <--

Reply:
Fixed.